pkg/types/bundles.go

```diff
@@ -0,0 +1,17 @@
+package types
+
+import (
+	"time"
+
+	"github.com/lib/pq"
+)
+
+type ExecuteBundle struct {
+	ID          int64          `db:"id"`
+	TxHashes    pq.StringArray `db:"tx_hashes"` // Use pq.StringArray instead of []string
```

Comment:
If you are using this for an API, pls consider to use `[]string`

Reply:
type pq.StringArray []string

Reply:
this type is actually string slice so it still backward compatible imo